### DISCUSSION TITLE
增加特定网址原文语言的设定

### DISCUSSION THREAD

**Discussion by Xinjian-Zhang** [immersive-translate/immersive-translate#1248] [Ideas]
### 新功能 | New Feature

1. 对于特定网址，增加一个原文设定

### 为什么您想要添加这些功能 | Why you want to add these features

1. 由于一些网页代码的不规范或缺失，没有标注网站的语言，原文语言会识别为英文，因此每次访问需要重新手动选择原文语言，很麻烦

### 提议 | Proposal

和“总是翻译该网站”一样，增加一个字段记录该网页的语言代码即可

**Comment by theowenyoung** [comment]
目前默认行为是，你指定一个语言，那么会记住这件事，该网页以后打开也会认为是这个语言。

如果想要弄网站级别的，那么可能要自己修改 开发者 设置里面的自定义配置里，大概是这样：

```json
  "sourceLanguageUrlPattern": {
    "en": {
      "matches": ["*.google.com"]
    }
  }
```
